kubernetes/reference/adoc/SUSE/SLES/SA_vars.adoc

```diff
@@ -13,4 +13,22 @@
 :pn_SLES_AutoYaSTDocURL: https://documentation.suse.com/sles/15-SP3/single-html/SLES-autoyast/#book-autoyast
 
 :pn_SLES_VirtDocURL: https://documentation.suse.com/sles/15-SP3/single-html/SLES-virtualization/#book-virt
+:pn_SLES_LibvirtTerraformURL: https://registry.terraform.io/providers/dmacvicar/libvirt/latest/docs
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                                                                               
+~                        
```

Comment:
Are these dashes here placeholders?

Reply:
Was shocked to see this and congrats for finding the main issue for the builds.  This is remedied in main already